docs/reference/NuGet-Client-SDK.md

```diff
@@ -10,9 +10,30 @@ ms.topic: conceptual
 # NuGet Client SDK
 
 The *NuGet Client SDK* refers to a group of NuGet packages:
-
-* [`NuGet.Protocol`](https://www.nuget.org/packages/NuGet.Protocol) - Used to interact with HTTP and file-based NuGet feeds
-* [`NuGet.Packaging`](https://www.nuget.org/packages/NuGet.Packaging) - Used to interact with NuGet packages. `NuGet.Protocol` depends on this package
+* [`NuGet.CommandLine`](https://www.nuget.org/packages/NuGet.CommandLine) - NuGet Command Line Interface.
+* [`NuGet.Indexing`](https://www.nuget.org/packages/NuGet.Indexing) - NuGet's indexing library for the Visual Studio client search functionality.
+* [`NuGet.VisualStudio.Contracts`](https://www.nuget.org/packages/NuGet.VisualStudio.Contracts) - This package contains RPC contracts for NuGet’s Visual Studio Service Broker extensibility APIs.
+* [`NuGet.VisualStudio`](https://www.nuget.org/packages/NuGet.VisualStudio) - This package provides APIs for invoking NuGet services in Visual Studio.
+* [`Microsoft.Build.NuGetSdkResolver`](https://www.nuget.org/packages/Microsoft.Build.NuGetSdkResolver) - MSBuild SDK resolver for NuGet packages.
+* [`NuGet.Build.Tasks.Console`](https://www.nuget.org/packages/NuGet.Build.Tasks.Console) - NuGet Build tasks for MSBuild and dotnet restore.
+* [`NuGet.Build.Tasks.Pack`](https://www.nuget.org/packages/NuGet.Build.Tasks.Pack) - NuGet tasks for MSBuild and dotnet pack.
+* [`NuGet.Build.Tasks`](https://www.nuget.org/packages/NuGet.Build.Tasks) - NuGet tasks for MSBuild and dotnet restore.
+* [`NuGet.CommandLine.XPlat`](https://www.nuget.org/packages/NuGet.CommandLine.XPlat) - NuGet executable wrapper for the dotnet CLI nuget functionality.
+* [`NuGet.Commands`](https://www.nuget.org/packages/NuGet.Commands) - Complete commands common to command-line and GUI NuGet clients.
+* [`NuGet.Common`](https://www.nuget.org/packages/NuGet.Common) - Common utilities and interfaces for all NuGet libraries.
+* [`NuGet.Configuration`](https://www.nuget.org/packages/NuGet.Configuration) - NuGet's configuration settings implementation.
+* [`NuGet.Credentials`](https://www.nuget.org/packages/NuGet.Credentials) - NuGet client's authentication models.
+* [`NuGet.DependencyResolver.Core`](https://www.nuget.org/packages/NuGet.DependencyResolver.Core) - NuGet's PackageReference dependency resolver implementation.
+* [`NuGet.Frameworks`](https://www.nuget.org/packages/NuGet.Frameworks) - NuGet's understanding of target frameworks.
+* [`NuGet.LibraryModel`](https://www.nuget.org/packages/NuGet.LibraryModel) - NuGet's types and interfaces for understanding dependencies.
+* [`NuGet.Localization`](https://www.nuget.org/packages/NuGet.Localization) - NuGet localization package.
+* [`NuGet.PackageManagement`](https://www.nuget.org/packages/NuGet.PackageManagement) - NuGet Package Management functionality for Visual Studio installation flow.
+* [`NuGet.Packaging.Core`](https://www.nuget.org/packages/NuGet.Packaging.Core) - The (former home to) core data structures for `NuGet.Packaging`.
```

Comment:
As the description points out, this package no longer contains anything. It's entirely superceded by NuGet.Packaging. Unrelated to this PR, we've had the type forwarding for so long now, we can probably stop creating and publishing this package

Reply:
Yep, I want us to remove that package: https://github.com/NuGet/Home/issues/12495. 

I have it as a breaking change.